src/main/java/com/gorlah/kappabot/function/command/CommandFunction.java

```diff
@@ -0,0 +1,34 @@
+package com.gorlah.kappabot.function.command;
+
+import com.gorlah.kappabot.function.BotRequestMetadata;
+import com.gorlah.kappabot.command.CommandPayloadBuilder;
+import com.gorlah.kappabot.command.CommandProcessor;
+import com.gorlah.kappabot.function.BotFunction;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class CommandFunction implements BotFunction {
```

Comment:
nice

Reply:
Thanks. I felt pretty good about this one.